### DISCUSSION TITLE
Use of displaced mesh within transfers and SAM

### DISCUSSION THREAD

**Discussion by robertjdemuth** [idaholab/moose#25604] [Q&A General]
I am using MOOSE's MultiApp feature to model the 3D conduction of a fuel rod with a 1D coolant channel in SAM. I am having a hard time understanding the purpose of the displaced mesh within SAM and the transfers. Most example files I have been able to find have use_displaced=true within the SAM subapp output section, but don't have an explanation as to why. 
![image](https://github.com/idaholab/moose/assets/145687537/588f1906-83b7-4361-b685-3d811c3da622)
The transfers section also has a displaced_source_mesh and displaced_target_mesh option.
![image](https://github.com/idaholab/moose/assets/145687537/51030052-f640-414a-8bb5-18a0bbde0fc2)
I have run the code with displaced meshes on and off and get drastically different results for my heat transfer coefficient, none of which seem correct.
displaced mesh off:
![image](https://github.com/idaholab/moose/assets/145687537/813dd33b-69c2-49f3-9705-cb5ad4229b1d)
displaced mesh on for subapp, on for master:
![image](https://github.com/idaholab/moose/assets/145687537/16f38969-902c-4568-9b3e-d0fbd75abf6e)
I am just modelling diffusion and never define a displaced mesh, so I am not sure why turning on displaced mesh makes a difference. Any advice would be appreciated.

Edit:
My heat transfer coefficient results are stripped. I'm thinking this has something to do with the SAM built in displaced mesh defined wrong on my end. Any advice would be appreciated.
![image](https://github.com/idaholab/moose/assets/145687537/37a976f9-4cf5-4409-bf2d-eca7f687701e)

**Comment by GiudGiud** [comment]
If I remember well,

SAM uses the mesh displacements to have the components be at their real location in space iirc
if you do not use that, they are probably effectively all on the same axis starting from the origin when the transfer seeks to compute the nodal locations of SAM components

**Reply by robertjdemuth** [reply]
This makes sense, thanks. I changed the location of the SAM component to confirm.

**Comment by lindsayad** [comment]
@rui-hu @lingzou
